evalutils/templates/evaluator/{{ cookiecutter.package_name }}/Dockerfile

```diff
@@ -16,6 +16,12 @@ ENV PATH="/home/evaluator/.local/bin:${PATH}"
 
 RUN python -m pip install --user -U pip
 
+{% if cookiecutter.dev_build|int -%}
```

Comment:
Wait, does this mean that we can test without the existance of the dev branch? If so, super!

Reply:
yes that is what it means